wildlifelicensing/apps/applications/views/conditions.py

```diff
@@ -51,7 +51,9 @@ def get_context_data(self, **kwargs):
     def post(self, request, *args, **kwargs):
         application = get_object_or_404(Application, pk=self.args[0])
 
-        application.processing_status = 'ready_to_issue'
+        current_status = application.processing_status
+        next_status = 'ready_to_issue' if current_status not in ['issued'] else current_status
+        application.processing_status = next_status
```

Comment:
I was waiting for your comment. I did it on purpose because I think we should also add the 'declined' in the list. What do you think.